plugins/widget/plugin.js

```diff
@@ -1420,6 +1420,21 @@
 			this.editor.focus();
 		},
 
+		/**
+		 * Refreshes widget's mask. Can be used together with {@link CKEDITOR.plugins.widget#refreshParts} method to reinitialize mask
+		 * for dynamically created widgets.
+		 */
+		refreshMask: function() {
+			setupMask( this );
+		},
+
+		/**
+		 * Reinitializes widget's {@link CKEDITOR.plugins.widget#parts parts}.
```

Comment:
Some broader explanation could be useful, e.g. that it covers the case when widget HTML is async or modified during widget lifecycle and this method can be used to link new DOM elements to `parts`. Also that it covers only parts which were not initialized before (so with absent element on widget initialization).